app/src/main/java/ch/logixisland/anuto/engine/render/Renderer.java

```diff
@@ -60,6 +62,22 @@ public void invalidate() {
         }
     }
 
+    public Bitmap getScreenshot() {
+        RectF mapRect = mViewport.getMapRect();
+        Bitmap bitmap = Bitmap.createBitmap(Math.round(mapRect.width()), Math.round(mapRect.height()), Bitmap.Config.ARGB_8888);
+        Canvas canvas = new Canvas(bitmap);
+
+        //that call would eventually end in Renderer.draw(), so just take a shortcut.
+        //it should be sufficient, since we only need our drawables
+        //mViewRef.get().draw(canvas);
+
+        //perhaps build another draw routine, so the game map is better visible
+        //e.g. an independent theme just for savegame screenshot
+        draw(canvas);
```

Comment:
I actually like this approach very much!

Reply:
Thanks. And I am serious in my comment.
I already tried some stuff, but never really got satisfied.

A forth, neutral theme would have the positiv effect, that the load game menu would look homogeneous, no matter with which theme the game runs in the meantime.

Of course, beside the aim for a better screenshot visualization of the game map...